deploy/docker/common.properties

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
Can we just reuse this file instead of duplicating it, we have to maintain several `common.properties` files... https://github.com/apache/dolphinscheduler/blob/6466cc7c41bc32596499b023ed483387cb32d02a/dolphinscheduler-common/src/main/resources/common.properties#L17

Reply:
> Can we just reuse this file instead of duplicating it, we have to maintain several `common.properties` files...
> 
> https://github.com/apache/dolphinscheduler/blob/6466cc7c41bc32596499b023ed483387cb32d02a/dolphinscheduler-common/src/main/resources/common.properties#L17

+1